cves/CVE-2011-4136.yml

```diff
@@ -15,7 +15,7 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
```

Comment:
I give this 2 upvotes. An interesting vulnerability. Though I think its a niche one.